addons/mail/static/src/discuss/call/common/infinite_mirroring_warning.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="discuss.InfiniteMirroringWarning">
+        <div class="d-flex justify-content-start align-items-start position-fixed top-0 start-0 w-100 h-100 pe-none">
+            <div class="d-flex bg-dark rounded justify-content-between text-white fs-3 w-100 m-5 p-3 pe-auto">
+                <div>
+                    Please share a specific window or tab or another monitor to avoid the infinite mirror effect.
+                </div>
+                <div class="d-flex gap-2">
+                    <button t-on-click="()=>this.props.onClose()" class="btn btn-primary pe-auto">
+                        Ok
+                    </button>
+                    <button t-on-click="()=>this.props.onClose(true)" class="btn btn-secondary pe-auto">
+                        Ignore
+                    </button>
```

Comment:
`Ok` should keep screen-sharing.
The `ignore` should be renamed to `Stop sharing`.